spec/message.ebnf

```diff
@@ -1,25 +1,20 @@
-Message ::= Declaration* ( Pattern | Selector Variant+ )
+Message ::= (s? Declaration)* s? Body s?
```

Comment:
```suggestion
Message ::= s? (Declaration s)* Body s?
```
I think we should require a space after each declaration. Without that requirement, we would allow for e.g.:
```
let $foo={(foo)}{body}
```
or
```
let $foo={(foo)}let $bar={(bar)}{body}
```